From 169346b6ebac49265b19a91c02b5936610794dee Mon Sep 17 00:00:00 2001
From: Bruno <brunosmmm@gmail.com>
Date: Mon, 27 Mar 2023 13:47:00 -0400
Subject: [PATCH] Import namespace

---
 driver/axi_dma.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/driver/axi_dma.c b/driver/axi_dma.c
index 1a2f2e9..e8d014b 100644
--- a/driver/axi_dma.c
+++ b/driver/axi_dma.c
@@ -134,3 +134,4 @@ MODULE_LICENSE("GPL");
 MODULE_VERSION("1.0");
 MODULE_DESCRIPTION("Module to provide a userspace interface for transferring "
                    "data from the processor to the logic fabric via AXI DMA.");
+MODULE_IMPORT_NS(DMA_BUF);
-- 
2.39.2

